app/routes/account_details.js

```diff
@@ -0,0 +1,27 @@
+import Ember from "ember";
+import AuthorizeRoute from './authorize';
+
+export default AuthorizeRoute.extend({
+  model(params) {
+    var user = this.get('session.currentUser');
+    var organisationsUser = user.get('organisationsUsers') && user.get('organisationsUsers.firstObject');
+    var organisation = organisationsUser && organisationsUser.get('organisation');
+
+    return Ember.RSVP.hash({
+      organisation: params.orgId? this.store.peekRecord('gc_organisation', parseInt(params.orgId)) : organisation,
+      organisationsUser: organisationsUser,
+      user: user,
+      order: this.store.peekAll("order").filterBy("state", "draft")
```

Comment:
We have the feature that if there is no packages in order in `draft` state the `Order` gets deleted.

Reply:
I feel we have that feature that's fine but still in code we should try to avoid such case and change code as @Darshan-Chauhan suggested. :)

Let me know your thoughts.

CC: @Darshan-Chauhan